.github/workflows/tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           cache: 'npm'
 
       - name: Install npm dependencies
-        run: npm ci
+        run: npm ci --force
```

Comment:
I'm putting this in temporary to get around the npm issue not liking betas in deps